docs-win/index.md

```diff
@@ -0,0 +1,297 @@
+```
+.\Scripts\ezbeq.exe
+```
+![ezbeq error](./img/18.png)
+
+5\. Open Windows Explorer (file manager), navigate to the user home folder, and then double-click on the .ezbeq folder.
+
+![Windows explorer](./img/19.png)
+
+6\. Double-click on the ezbeq.yml file.
+
+![Windows explorer .ezbeq folder](./img/20.png)
+
+7\. Double-click on the file, choosing Notepad to open the file (or the preferred text editor).
+
+![Unknown file dialog](./img/21.png)
+
+![Unknown file dialog2](./img/22.png)
+
+8\. Initial configuration file example
+
+![Example of config file](./img/23.png)
+
+9\. Host will be the name of the system. Change the "minidspExe" line to the location where the minidsp.exe was placed initially. Save the file.
+
+NOTE: Ensure that .exe is added to the end as well, or it won't work correctly.
+
+![ezbeq configuration file update](./img/24.png)
+
+10\. Switch back to the Powershell window, press the up arrow, and restart ezbeq (or type `.\Scripts\ezbeq.exe`). This time it should stay running (the prompt will not return). If it errors again, the mostly reason is that ezbeq cannot find the minidsp executable.
+
+![Run ezbeq again](./img/25.png)
+
+11\. A Windows Defender window may pop up. Select the "Private networks" option. Uncheck the "Public networks". Click on Allow Access.
+
+![Windows Defender popup](./img/26.png)
+
+10\. ezbeq runs on port 8080 of the system by default. Open a web browser and connect to the system on port 8080. Note that the web server is not encrypted, so the URL must be entered as http, otherwise most current browsers will automatically try to connect using HTTPS (secured) and fail.
+
+Open a browser window to `http://127.0.0.1:8080`
+
+(127.0.0.1 is a special reference to the local machine. If the server running ezbeq is remote, enter the IP of it instead of 127.0.0.1.)
+
+![Example of ezbeq interface](./img/27.png)
+
+11\. To load a BEQ onto the MiniDSP 2x4HD, select a title (search is available in the upper right corner), then click on the &quot;up arrow&quot; on the corresponding config slot to be used.
+
+![Example of selecting a title and uploading to config slot 2](./img/28.png)
+
+13\. If the update is successful, the title will be display in the &quot;Loaded&quot; column. Otherwise, &quot;ERROR&quot; will be displayed.
+
+![Example of successful title load](./img/29.png)
+
+14\. To clear a title from a config slot, click on the "X".
+
+15\. To switch configuration slots without changing the title loaded, click on the play icon.
+
+### Automatically launch ezbeq on boot
+
+These steps are optional but are highly recommended. If these steps are not followed, ezbeq will need to be started manually after every reboot of the Windows device, or if the Powershell window is closed.
```

Comment:
does it leave a cmd window open when this runs? if so, you may want to use `nircmd exec hide` to get rid of it

Reply:
It did not leave a window open when starting on boot, no. But if the user manually started it from the Powershell window, then yes, the PS window stays open. Is that what you're asking?